tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Great openeing!</p><p> </p><p>As building an oracle is THE best example for the usefulness of high-level functional design, we want to emphasize it in the terminology. Please change the "construct an oracle in a quantum circuit" to "construct and design an oracle for a quantum algorithm?" or "construct and design an in practice"</p><p> </p><p>Then, I would explain a bit more the XOR operation and that the function is a binary function with a binary output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>